src/components/core/geoChart/tests/geoChartDataLayers.spec.ts

```diff
@@ -23,7 +24,12 @@ describe("createPushpinDataLayer", () => {
                 data: [],
             },
         };
-        const layer: mapboxgl.Layer = createPushpinDataLayer(dataSourceName, geoData, mapboxConfig);
+        const layer: mapboxgl.Layer = createPushpinDataLayer(
+            dataSourceName,
+            geoData,
+            mapboxConfig,
+            DEFAULT_COLORS[0],
```

Comment:
is this the same with `DEFAULT_PUSHPIN_COLOR_VALUE`?

Reply:
fixed